src/index.html

```diff
@@ -9,10 +12,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
SCSS files should not be linked directly in HTML. Instead, compile your SCSS to CSS and link the CSS file.